src/lib/select/select.ts

```diff
@@ -890,31 +893,29 @@ export class MdSelect extends _MdSelectMixinBase implements AfterContentInit, On
 
   /** Calculates the scroll position and x- and y-offsets of the overlay panel. */
   private _calculateOverlayPosition(): void {
+    this._triggerHeight = this.trigger.nativeElement.getBoundingClientRect().height;
+    this._triggerFontSize = this._measureFontSizeEl.nativeElement.getBoundingClientRect().height;
```

Comment:
I'm pretty sure that you'll get a pixel value no matter what. Should be pretty easy to verify. As for performance, it should be pretty close to the `getBoundingClientRect` call.